packages/@lwc/engine-core/src/framework/wiring.ts

```diff
@@ -188,23 +195,12 @@ function createConnector(vm: VM, name: string, wireDef: WireDef): WireAdapter {
     };
 
     // Computes the current wire config and calls the update method on the wire adapter.
-    // This initial implementation may change depending on the specific wire instance, if it has params, we will need
-    // to observe changes in the next tick.
-    let computeConfigAndUpdate = () => {
-        updateConnectorConfig(configCallback(component));
-    };
-
-    if (hasDynamicParams) {
```

Comment:
no need to differentiate any longer: the config won't be computed until the component is connected.